src/resources/storefront/websites-resource.js

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
Hey @justnero and @mattsanf  
The SDK auto-generator used `WebsitesResource` instead of `WebsiteResource` based on the storefront api definition format: 

https://github.com/Rebilly/api-definitions/blob/master/openapi/paths/storefront/websites%40%7Bid%7D.yaml

In order not to break the public contract, the resource is still exposed as website (singular).

Is that ok?